packages/desktop-client/src/style/styles.ts

```diff
@@ -103,31 +104,35 @@ let hiddenScrollbars = false;
 // lightScrollbar => primary
 // darkScrollbar => secondary
 function onScrollbarChange() {
-  styles.lightScrollbar = !hiddenScrollbars && {
-    '& ::-webkit-scrollbar': {
-      width: 11,
-      backgroundColor: 'rgba(200, 200, 200, .2)',
-    },
-    '& ::-webkit-scrollbar-thumb': {
-      width: 7,
-      borderRadius: 30,
-      backgroundClip: 'padding-box',
-      border: '2px solid rgba(0, 0, 0, 0)',
-    },
-    '& ::-webkit-scrollbar-thumb:vertical': {
-      backgroundColor: '#d0d0d0',
-    },
-  };
+  styles.lightScrollbar = {
+    ...(!hiddenScrollbars && {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏object spread is not necessary here

Reply:
Same as my previous comment

Reply:
I think with your latest type change we can now revert the changes in this file